java/java.lsp.server/vscode/src/nbcode.ts

```diff
@@ -75,7 +75,7 @@ export function launch(
     }
     ideArgs.push(...extraArgs);
     
-    if (env['netbeans_debug'] && extraArgs && extraArgs.find(s => s.includes("--list"))) {
+    if (extraArgs && extraArgs.find(s => s.includes("--list"))) {
```

Comment:
Why was the env variable condition removed ?

Reply:
Oops, this is a mistake, will fix.